### ISSUE TITLE
loginSucceeded() Signal doesn't work

### ISSUE THREAD

**Issue by syrupderg** [sddm/sddm#1960]

**Comment by Vogtinator** [comment]
?

**Comment by syrupderg** [comment]
> ?

no matter what i do, the loginSucceeded() signal just doesnt work, i tested with other themes and they all also have this problem. maybe because when the user logs in, sddm-greeter gets killed instantly before the loginSucceeded() function could do stuff

**Comment by Vogtinator** [comment]
Yeah, kind of. With rootful X11, the greeter has to be stopped as soon as possible anyway. For the others, it's technically possible to have it open for longer, but not that useful. What probably triggers it is this:

```
        connect(m_proxy, &GreeterProxy::loginSucceeded, view, &QQuickView::close);
```

**Comment by Vogtinator** [comment]
The behavior can probably stay that way, just the documentation could use some adjustment.

**Comment by syrupderg** [comment]
> Yeah, kind of. With rootful X11, the greeter has to be stopped as soon as possible anyway. For the others, it's technically possible to have it open for longer, but not that useful. What probably triggers it is this:
> 
> ```
>         connect(m_proxy, &GreeterProxy::loginSucceeded, view, &QQuickView::close);
> ```

can i delay it, for example, "x" seconds?

**Comment by syrupderg** [comment]
![image](https://github.com/sddm/sddm/assets/67545942/5b6b6eea-34a7-40dc-8f48-8b76f84e155c)
i found this message from breeze sddm theme, i hope there is a way to fix this bug because for me, this function straight up doesnt do anything so i cant even delay it since sddm greeter gets killed before the function can do anything
